src/lib/application/schema.json

```diff
@@ -45,6 +45,15 @@
       "type": "string",
       "description": "Nest application package manager."
     },
+    "platform": {
+      "type": "string",
+      "description": "Nest application underlying platform to use.",
+      "x-prompt": {
```

Comment:
Adding this prompt could be seen as a breaking change? I mean, some users may be using Nest CLI on scripts that are not expecting another prompt. If someone agree with me, I'll drop this `x-prompt` prop.